Lib/pathlib.py

```diff
@@ -23,6 +24,12 @@
 # Internals
 #
 
+_WIN_RESERVED_NAMES = frozenset(
+    {'CON', 'PRN', 'AUX', 'NUL', 'CONIN$', 'CONOUT$'} |
+    {'COM%s' % c for c in '123456789\xb9\xb2\xb3'} |
+    {'LPT%s' % c for c in '123456789\xb9\xb2\xb3'}
```

Comment:
Might make a tiny difference on module import speed, f-strings are faster
```suggestion
    {f'COM{c}' for c in '123456789\xb9\xb2\xb3'} |
    {f'LPT{c}' for c in '123456789\xb9\xb2\xb3'}
```